### ISSUE TITLE
Where to place video files

### ISSUE THREAD

**Issue by reillydonovan** [magdesign/PocketVJ#2]
I went through all of the apt-get -y installs and copied all of the files from PocketVJ-master to /var/www.  Now when I go to 10.0.0.4 from another cpu on the network I get the expected interface, however when I click on open a file, it tries to redirect me to eXtplorer/index.php but returns 403 - forbidden.  I have index.php in that directory, seems like some kind of access problem.  Would you have any suggestions?  Thank you so much!

**Comment by magdesign** [comment]
Did you make the whole directory /var/www executable with chmod 755 ?
The video files go into: /media/internal/

Best is to go to the Display tab and hit: Get Resolution, if you get a feedback, things are gonne be alright.

**Comment by reillydonovan** [comment]
Wow thanks a lot!  That totally worked!  Amazing!

**Comment by reillydonovan** [comment]
So now the only problem I'm having is actually getting some playback or a test screen to display... I copied the config.txt file to my pi and can access eXtplorer from my mac's browser.  I have the two test files in the media/interfaces folder.  I do get output printed to the output console when I click start and stop, but nothing happens.  When I go to display nothing gets returned.  Any thoughts?  Thanks again!

**Comment by barlaensdoonn** [comment]
I'm having the same issue. I can't seem to control anything from the interface - display, reboot, test screen, etc. all print to the console, but nothing is actually done on the pi.

**Comment by magdesign** [comment]
Did you configure /etc/lighttpd/lighttpd.conf ?
Seems like your pi does not allow to execute .php commands from the webserver.
Maybe there are some security changes on the newer kernel, since im still running kernel 3.10.24+
and lighttpd/1.4.31 (I will try with a newer raspian)

**Comment by reillydonovan** [comment]
How do you configure the .conf file?

**Comment by magdesign** [comment]
sudo nano /etc/lighttpd/lighttpd.conf

should look something like this:

server.modules = (
        "mod_access",
        "mod_alias",
        "mod_compress",
        "mod_redirect",
#       "mod_rewrite",
)

server.document-root        = "/var/www"
server.upload-dirs          = ( "/var/cache/lighttpd/uploads" )
server.errorlog             = "/var/log/lighttpd/error.log"
server.pid-file             = "/var/run/lighttpd.pid"
server.username             = "www-data"
server.groupname            = "www-data"
server.port                 = 80


index-file.names            = ( "index.php", "index.html", "index.lighttpd.html" )
url.access-deny             = ( "~", ".inc" )
static-file.exclude-extensions = ( ".php", ".pl", ".fcgi" )

compress.cache-dir          = "/var/cache/lighttpd/compress/"
compress.filetype           = ( "application/javascript", "text/css", "text/html", "text/plain" )

# default listening port for IPv6 falls back to the IPv4 port
include_shell "/usr/share/lighttpd/use-ipv6.pl " + server.port
include_shell "/usr/share/lighttpd/create-mime.assign.pl"
include_shell "/usr/share/lighttpd/include-conf-enabled.pl"

**Comment by magdesign** [comment]
And I figured out why you could not execute any commands from the webinterface.
You have to add the www-data user to the sudoers list! 
make visudo and scroll down to the button, add this line: 


www-data ALL=(ALL) NOPASSWD: ALL

**Comment by barlaensdoonn** [comment]
Adding www-data to sudoers list worked!!! The lighttpd.conf was actually already configured as listed, didn't need to change it, just had to execute visudo and add the one line.

Thank you so much!

**Comment by lcsmedina** [comment]
Hello

I'm trying to make a pocketvj from scratch.
I followed all the instructions, and I think they went well.
I can not see the control panel. I try to open from google chrome the ip address of the raspberry pi (in my case 192.168.0.22).

Before modifying the /etc/lighttpd/lighttpd.conf  /var/www/html  by  /var/www,
I could see the /var/www/html/index.lighttpd.html in google chrome. But now changing the file again and returning it to what it was, but nothing happens.

I already did:
visudo --> www-data ALL = (ALL) NOPASSWD: ALL
sudo lighty-enable-mod fastcgi-php
chmod 755 -R / var / www
sudo /etc/init.d/lighttpd force-reload

The raspberry pi is using a wired connection to the router. The ip of the router is 192.168.0.1

ls of the www folder
root@raspberrypi:/var/www# ls
05_mapper.png AUTHORS.md LICENSE.md PocketVJ-CP-v3 README.md RELEASE_NOTES.md ROADMAP.md backend.php css eXtplorer favicon.ico fonts html images index.html js offline.min.js projector_passwords.md streamer sync themes

eXtplorer does not walk either

thank you very much!!
